codegen/src/main/java/software/amazon/awssdk/codegen/emitters/tasks/AwsGeneratorTasks.java

```diff
@@ -33,7 +33,8 @@ public AwsGeneratorTasks(GeneratorTaskParams params) {
     }
 
     private Iterable<GeneratorTask> createAwsTasks(GeneratorTaskParams params) {
-        return new CompositeIterable<>(new AsyncClientGeneratorTasks(params));
+        return new CompositeIterable<>(new AsyncClientGeneratorTasks(params),
```

Comment:
Can you add a TODO for the async generator tasks. It's something we'll defintely wanna push up to the generic tasks.

Reply:
Done